posts/2024-03-21-Rust-1.77.0.md

```diff
@@ -0,0 +1,124 @@
+debuginfo, which means that statically linked results would include the
+debuginfo from the standard library even if the local compilations didn't
+explicitly request debuginfo.
+
+Users which do want debuginfo can explicitly enable it with the
+[debug](https://doc.rust-lang.org/stable/cargo/reference/profiles.html#debug)
+flag in the relevant Cargo profile.
+
+### Clippy adds a new `incompatible_msrv` lint
+
+The Rust project only supports the latest stable release of Rust. Some
+libraries aim to have an older minimum supported Rust version (MSRV), typically
+verifying this support by compiling in CI with an older release. However, when
+developing new code, it's convenient to use latest documentation and the latest
+toolchain with fixed bugs, performance improvements, and other improvements.
+This can make it easy to accidentally start using an API that's only available
+on newer versions of Rust.
+
+Clippy has added a new lint, [`incompatible_msrv`](https://rust-lang.github.io/rust-clippy/master/index.html#/incompatible_msrv),
+which will inform users if functionality being referenced is only available on
+newer versions than their
+[declared MSRV](https://github.com/rust-lang/rust-clippy/?tab=readme-ov-file#specifying-the-minimum-supported-rust-version).
+
+### Stabilized APIs
+
+- [`array::each_ref`](https://doc.rust-lang.org/stable/std/primitive.array.html#method.each_ref)
+- [`array::each_mut`](https://doc.rust-lang.org/stable/std/primitive.array.html#method.each_mut)
+- [`core::net`](https://doc.rust-lang.org/stable/core/net/index.html)
+- [`f32::round_ties_even`](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.round_ties_even)
+- [`f64::round_ties_even`](https://doc.rust-lang.org/stable/std/primitive.f64.html#method.round_ties_even)
+- [`mem::offset_of!`](https://doc.rust-lang.org/stable/std/mem/macro.offset_of.html)
+- [`slice::first_chunk`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.first_chunk)
+- [`slice::first_chunk_mut`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.first_chunk_mut)
+- [`slice::split_first_chunk`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.split_first_chunk)
+- [`slice::split_first_chunk_mut`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.split_first_chunk_mut)
+- [`slice::last_chunk`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.last_chunk)
+- [`slice::last_chunk_mut`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.last_chunk_mut)
+- [`slice::split_last_chunk`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.split_last_chunk)
+- [`slice::split_last_chunk_mut`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.split_last_chunk_mut)
+- [`slice::chunk_by`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.chunk_by)
+- [`slice::chunk_by_mut`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.chunk_by_mut)
```

Comment:
No docs found for this link. Or link is not working!

Reply:
slice::chunk_by_mut

Reply:
Both of these will work once the stable release goes out.

Reply:
Renamed during stabilization, you can see the correct doc in nightly. Once we release stable 1.77 the stable doc will be rebuilt, so it will become valid link later and nothing wrong here.